invenio_rdm_records/resources/resources.py

```diff
@@ -43,6 +44,22 @@ def p(route):
     #
     # Review request
     #
+    @request_search_args
+    @request_view_args
+    @response_handler(many=True)
+    def search_community_records(self):
```

Comment:
It does not look like that you have created a route for this endpoint, and there is no test for this resource. The endpoint cannot be used.
The URL, I think, should be `/communities/<pid_value>/records`